utils/notesCore.ts

```diff
@@ -39,7 +34,7 @@ export class NoteManagerCore {
 	) {
 		this.citationStyleKind = citationStyleKind;
 		this.citationInlineStyleKind = citationInlineStyleKind;
-		this.notes = notes;
+		this.notes = { ...notes };
```

Comment:
I don't understand what spreading into brackets does. Creates new object instead of just copying a reference to the same object?

Reply:
Exactly — I noticed that the note manager was actually modifying the `initialData` object, which made debugging a little difficult. Not entirely necessary but good hygiene.